testsuite/manualmode/src/test/java/org/wildfly/core/test/standalone/mgmt/PreparedResponseTestCase.java

```diff
@@ -98,7 +100,16 @@ public void reloadServer() throws Exception {
         try (ManagementClient managementClient = getManagementClient()) {
             block(managementClient);
             long timeout = SHUTDOWN_WAITING_TIME + System.currentTimeMillis();
-            managementClient.executeForResult(Operations.createOperation(RELOAD, new ModelNode().setEmptyList()));
+            try {
+                ModelNode result = managementClient.getControllerClient().execute(
+                        Operations.createOperation(RELOAD, new ModelNode().setEmptyList()));
+                Assert.assertEquals("success", result.get(OUTCOME).asString());
+            } catch (IOException e) {
+                final Throwable cause = e.getCause();
+                if (!(cause instanceof ExecutionException) && !(cause instanceof CancellationException)) {
+                    throw new RuntimeException(e);
+                } // else ignore, this might happen if the channel gets closed before we got the response
+            }
```

Comment:
Thanks for this. We have a util class, ServerReload, that should be used for this kind of thing. It looks like moving this test to it was missed. The block here looks the same as ServerReload.executeReload(managementClient.getControllerClient(), false) so please use that instead.

I thought it might be, but the ServerReload.executeReloadAndWaitForCompletion call is not appropriate here as this test is asserting some intermediate state while awaiting completion.